x-pack/plugins/security_solution/public/management/components/console/components/command_execution_output.tsx

```diff
@@ -8,6 +8,7 @@
 import React, { memo, useCallback, useMemo } from 'react';
 import { EuiLoadingChart } from '@elastic/eui';
 import styled from 'styled-components';
+import { CommandExecutionComponentProps } from '../types';
```

Comment:
Let's state explicitly here and elsewhere that types are imported `import type { CommandExecutionComponentProps }...`

Reply:
👍 

_(I wonder why eslint is not catching this - it catches it in some other directories. My guess is that its turned off for some areas fo security solution 🤷‍♂️ )